### ISSUE TITLE
Problem with jcenter

### ISSUE THREAD

**Issue by ObidosDev** [launchdarkly/react-native-client-sdk#108]
**Describe the bug**

**To reproduce**
Just build app for android from the source code

**Expected behavior**
Build is OK

**Logs**
```
A problem occurred configuring project ':launchdarkly-react-native-client-sdk'.
> Could not resolve all artifacts for configuration ':launchdarkly-react-native-client-sdk:classpath'.
   > Could not resolve com.android.tools:common:27.1.3.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools.build:manifest-merger:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android:zipflinger:4.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools.analytics-library:tracker:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:aaptcompiler:4.1.3 > com.android.tools.layoutlib:layoutlib-api:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.lint:lint-gradle-api:27.1.3 > com.android.tools.lint:lint-model:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3 > com.android.tools:dvlib:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3 > com.android.tools:repository:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3 > com.android.tools.analytics-library:shared:27.1.3
      > Could not resolve com.android.tools:common:27.1.3.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/common/27.1.3/common-27.1.3.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/common/27.1.3/common-27.1.3.pom'.
               > Read timed out
```

**SDK version**
"launchdarkly-react-native-client-sdk": "^5.1.0",

**OS/platform**
Android

**Comment by OrenMe** [comment]
same for me
`* What went wrong:
A problem occurred configuring project ':launchdarkly-react-native-client-sdk'.
> Could not resolve all artifacts for configuration ':launchdarkly-react-native-client-sdk:classpath'.
   > Could not resolve com.android.tools.build:bundletool:0.14.0.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3
      > Could not resolve com.android.tools.build:bundletool:0.14.0.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/bundletool/0.14.0/bundletool-0.14.0.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/android/tools/build/bundletool/0.14.0/bundletool-0.14.0.pom'.
               > Read timed out
   > Could not resolve com.google.test.platform:core-proto:0.0.2-dev.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3
      > Could not resolve com.google.test.platform:core-proto:0.0.2-dev.
         > Could not get resource 'https://jcenter.bintray.com/com/google/test/platform/core-proto/0.0.2-dev/core-proto-0.0.2-dev.pom'.
            > Could not GET 'https://jcenter.bintray.com/com/google/test/platform/core-proto/0.0.2-dev/core-proto-0.0.2-dev.pom'.
               > Read timed out
   > Could not resolve org.tensorflow:tensorflow-lite-metadata:0.1.0-rc1.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3
      > Skipped due to earlier error
   > Could not resolve it.unimi.dsi:fastutil:7.2.0.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-reflect:1.3.72.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.lint:lint-gradle-api:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3
      > Skipped due to earlier error
   > Could not resolve com.google.code.gson:gson:2.8.5.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > androidx.databinding:databinding-compiler-common:4.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.google.protobuf:protobuf-java-util:3.10.0
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools.build:manifest-merger:27.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3 > com.android.tools.analytics-library:shared:27.1.3
      > Skipped due to earlier error
   > Could not resolve org.glassfish.jaxb:jaxb-runtime:2.3.1.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > androidx.databinding:databinding-compiler-common:4.1.3
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdklib:27.1.3 > com.android.tools:repository:27.1.3
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.72.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.72.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72
      > Skipped due to earlier error
   > Could not resolve com.google.code.gson:gson:2.8.5.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta09
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.72.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta09
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta09
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.trove4j:trove4j:20160824.
     Required by:
         project :launchdarkly-react-native-client-sdk > com.android.tools.build:gradle:4.1.3 > com.android.tools.build:builder:4.1.3 > com.android.tools:sdk-common:27.1.3
      > Skipped due to earlier error`

**Comment by andrew-andilevko-itechart** [comment]
+1

**Comment by petr-laz** [comment]
+1 The same issue

**Comment by anton-patrushev** [comment]
+1

**Comment by louis-launchdarkly** [comment]
Hello @ObidosDev, @OrenMe, @andrew-andilevko-itechart, and @anton-patrushev, thank you all for reaching out and reporting the issue. The team had created the fix, and we will do the release for the fix soon.

I will post another update and close the issue when the fix is publicly available.

**Comment by gwhelanLD** [comment]
Hi everyone,

We've just released a [5.1.1 version](https://github.com/launchdarkly/react-native-client-sdk/releases/tag/5.1.1) of the SDK which updates the artifact repositories for the Android build, which should resolve this issue. Thanks to everyone for reaching out with this issue.

Thanks,
@gwhelanLD
